data/src/main/scala/ackcord/data/package.scala

```diff
@@ -127,6 +127,13 @@ package object data {
       c.getGuildChannel(guildId, channelId)
   }
 
+  type ForumTagId = SnowflakeType[ForumTag]
```

Comment:
Iffy on adding an alias for these unless they are used often. Not something I'd know well though